openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryV1.component.tsx

```diff
@@ -250,6 +250,7 @@ const GlossaryV1 = ({
     updatedDetails = {
       ...selectedData,
       displayName: displayName?.trim(),
+      name: displayName?.trim() || selectedData.name,
```

Comment:
Is this intentional? passing displayName for the name.

Reply:
yes!

Reply:
That's the main goal of PR :)